code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -474,18 +472,16 @@ func (fsh *StorageHandler) CommitWrite(ctx context.Context, r *http.Request) (*b
 	if err := writePreRedeem(ctx, allocationObj, &writeMarker, clientIDForWriteRedeem); err != nil {
 		return nil, err
 	}
-
 	err = connectionObj.ApplyChanges(ctx, writeMarker.AllocationRoot)
 	if err != nil {
 		return nil, err
 	}
-	rootRef, err := reference.GetReference(ctx, allocationID, "/")
+	rootRef, err := reference.GetReferenceHash(ctx, allocationID, "/")
```

Comment:
@abouroumine `connectionObj.ApplyChanges` can also return same rootHash. Can we do that?

Reply:
@abouroumine Did you consider this?